src/modules/dashboard/components/ColonyNavigation/ColonyNavigation.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ComponentProps, useMemo } from 'react';
+import { defineMessages } from 'react-intl';
+import { useParams } from 'react-router';
+
+import NavItem from './NavItem';
+
+const MSG = defineMessages({
+  linkTextActions: {
+    id: 'dashboard.ColonyNavigation.linkTextActions',
+    defaultMessage: 'Actions',
+  },
+  linkTextEvents: {
+    id: 'dashboard.ColonyNavigation.linkTextEvents',
+    defaultMessage: 'Events',
+  },
+  linkTextExtensions: {
+    id: 'dashboard.ColonyNavigation.linkTextExtensions',
+    defaultMessage: 'Extensions',
+  },
+  linkExtraExtensions: {
+    id: 'dashboard.ColonyNavigation.linkExtraExtensions',
+    defaultMessage: 'Coming Soon',
+  },
+});
+
+const displayName = 'dashboard.ColonyNavigation';
+
+const ColonyNavigation = () => {
+  const { colonyName } = useParams<{ colonyName: string }>();
+
+  // @TODO actually determine these
+  const hasNewActions = false;
+  const hasNewEvents = true;
+  const hasNewExtensions = false;
+
+  const items = useMemo<ComponentProps<typeof NavItem>[]>(
+    () => [
+      {
+        linkTo: `/colony/${colonyName}/actions`,
```

Comment:
Should this just be the colony home route, since Actions will be the default view?

So for example, should I just make this `/colony/${colonyName}` and then get rid of the new `COLONY_ACTIONS_ROUTE`?

Reply:
I think so yes, remove the "actions" route.

Whichever is the default selection for the side nav, should be just the colony route, and not have an individual one.

Alternatively, if we want to be clever, we can just redirect the "actions" route back to the colony route, so if somebody goes there by hand, it will be correctly redirected.